lib/team_builder.rb

```diff
@@ -44,16 +39,15 @@ def build_single_team(team_name)
   end
 
   def build_all_teams
-    static_config.map do |key, team_config|
+    static_config.map do |team_name, team_config|
```

Comment:
We could fix up `team_name` (rather than `key` to the commit where you first edit this line.